DemoApp/App.js

```diff
@@ -19,15 +19,9 @@ import {
   Button,
 } from 'react-native';
 
-import {
-  Colors,
-  DebugInstructions,
-  Header,
-  LearnMoreLinks,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
+import {Colors, Header} from 'react-native/Libraries/NewAppScreen';
 
-import Tracker from '@snowplow/react-native-tracker';
+import * as Snowplow from '@snowplow/react-native-tracker';
```

Comment:
As an example, I think it would be nicer to:

```suggestion
import { createTracker } from '@snowplow/react-native-tracker';
```